.github/workflows/release-to-candidate.yml

```diff
@@ -27,16 +27,16 @@ jobs:
         uses: snapcrafters/ci/get-architectures@main
 
   release:
-    name: 🚢 Release to latest/candidate
+    name: 🚢 Release to latest/lts
     needs: get-architectures
     runs-on: ubuntu-latest
-    environment: "Candidate Branch"
+    environment: "LTS Branch"
     strategy:
       matrix:
         architecture: ${{ fromJSON(needs.get-architectures.outputs.architectures-list) }}
     steps:
-      - name: 🚢 Release to latest/candidate
-        uses: snapcrafters/ci/release-to-candidate@main
+      - name: 🚢 Release to latest/lts
+        uses: snapcrafters/ci/release-to-lts@main
```

Comment:
This line is incorrect - the name of the action is still `snapcrafters/ci/release-to-candidate` :)

Reply:
Both at the same time. See https://github.com/snapcrafters/jenkins/issues/81 

I've opened an LTS track for the snap. See [here](https://forum.snapcraft.io/t/lts-track-request-for-jenkins-snap/39984).

Reply:
Okay, so in that case we need to update some workflows on the candidate branch too, as certain type of workflow (like scheduled workflows) will only run on a default branch

A good example of this is here:

- [prepare the candidate branch](https://github.com/snapcrafters/sublime-merge/commit/d879fcd9195ed3b94cb47d102bef686e0e0d785d)

Reply:
Once you correct this line so the action name is correct, this PR should be good to merge, then we just need another to update the candidate branch :)

Reply:
In the mean time, I've added the necessary credentials and protection rules for the LTS branch in readiness :) https://github.com/snapcrafters/tokenator/commit/fa0a7a8e6e475c2a2d4eec64c9517edba27f0a78

Reply:
I referenced the link above for prepping for a candidate branch. I had assumed it needed to be different to allow the automations to be delineated. I want to be sure that I'm understanding your comments and referenced link correctly - I need to change it from `lts` back to `candidate`?

Reply:
Yep, you just need need to change `uses: snapcrafters/ci/release-to-lts@main` back to `uses: snapcrafters/ci/release-to-candidate@main`

Reply:
No problem! Thanks!